_dev/src/components/product-feed/settings/shipping-settings/shipping-settings.vue

```diff
@@ -193,7 +194,19 @@ export default {
         });
     },
     disableContinue() {
-      return !this.carriers.every(validateDeliveryDetail);
+      const enabledCarriersLength = this.carriers.filter(
+        (carrier) => carrier.enabledCarrier).length;
+      const enabledCarriersCompletedLength = this.carriers.filter(
+        (carrier) => carrier.enabledCarrier
+      && validateHandlingTimes(carrier)
+      && validateTransitTimes(carrier)
+      && !!carrier.deliveryType).length;
+
+      if (enabledCarriersCompletedLength === enabledCarriersLength
+      && enabledCarriersCompletedLength > 0) {
+        return false;
+      }
+      return true;
```

Comment:
I smell duplicated code here, what about modifying the existing method `validateDeliveryDetail`?

Reply:
I was afraid to modify it because it's used at another place

Reply:
I let you check the other place we use it, but if the rules were the same, it may be needed to adapt the rules for it as well.